Assets/LeapMotion/Core/Scripts/Utils/Utils.cs

```diff
@@ -1039,6 +1230,253 @@ public struct ChildrenEnumerator : IEnumerator<Transform> {
+    }
+
+    #region Enumerators
+
+    /// <summary>
+    /// Slices numLines horizontal line Rects from this Rect and returns an enumerator that
+    /// will return each line Rect.
+    /// 
+    /// The height of each line is the height of the Rect divided by the number of lines
+    /// requested.
+    /// </summary>
+    public static HorizontalLineRectEnumerator TakeAllLines(this Rect r, int numLines) {
+      return new HorizontalLineRectEnumerator(r, numLines);
+    }
+
+    public struct HorizontalLineRectEnumerator : IQueryOp<Rect> {
+      Rect rect;
+      int numLines;
+      int index;
+
+      public HorizontalLineRectEnumerator(Rect rect, int numLines) {
+        this.rect = rect;
+        this.numLines = numLines;
+        this.index = -1;
+      }
+
+      public float eachHeight { get { return this.rect.height / numLines; } }
+
+      public Rect Current {
+        get { return new Rect(rect.x, rect.y + eachHeight * index, rect.width, eachHeight); }
+      }
+      public bool MoveNext() {
+        index += 1;
+        return index < numLines;
+      }
+      public HorizontalLineRectEnumerator GetEnumerator() { return this; }
+
+      public bool TryGetNext(out Rect t) {
+        if (MoveNext()) {
+          t = Current; return true;
+        }
+        else {
+          t = default(Rect); return false;
+        }
+      }
+
+      public void Reset() {
+        index = -1;
+      }
+
+      public QueryWrapper<Rect, HorizontalLineRectEnumerator> Query() {
+        return new QueryWrapper<Rect, HorizontalLineRectEnumerator>(this);
+      }
+    }
+
+    #endregion
+
+    #endregion
+
+    #region Pose Utils
+
+    /// <summary>
+    /// Returns the rotation that makes a transform at objectPosition point its forward
+    /// vector at targetPosition and keep its rightward vector parallel with the horizon
+    /// defined by a normal of Vector3.up.
+    /// 
+    /// For example, this will point an interface panel at a user camera while
+    /// maintaining the alignment of text and other elements with the horizon line.
+    /// </summary>
+    /// <returns></returns>
+    public static Quaternion FaceTargetWithoutTwist(Vector3 fromPosition, Vector3 targetPosition, bool flip180 = false) {
+      return FaceTargetWithoutTwist(fromPosition, targetPosition, Vector3.up, flip180);
+    }
+
+    /// <summary>
+    /// Returns the rotation that makes a transform at objectPosition point its forward
+    /// vector at targetPosition and keep its rightward vector parallel with the horizon
+    /// defined by the upwardDirection normal.
+    /// 
+    /// For example, this will point an interface panel at a user camera while
+    /// maintaining the alignment of text and other elements with the horizon line.
+    /// </summary>
+    public static Quaternion FaceTargetWithoutTwist(Vector3 objectPosition, Vector3 targetPosition, Vector3 upwardDirection, bool flip180 = false) {
```

Comment:
This method seems to behave exactly the same way as  Quaternion.LookRotation?

Reply:
It is basically another way to express Quaternion.LookRotation, I just wanted a more verbose syntax for it.

Reply:
If that's the case, why not just use Quaternion.LookRotation instead of all this extra math?  Would also probably good to mention in the docs that it does the same thing as Quaternion.LookRotation